src/components/Content/terminologies.json

```diff
@@ -1,16 +1,22 @@
 {
   "ahkaam": "rulings",
+  "asr": "afternoon prayer",
```

Comment:
```suggestion
  "asr": {
     "clarification": "afternoon prayer",
     "level": "basic"
   }
```

Reply:
Do it like this.
`level` represents the minimal level required to have a reasonable expectation that such a person is familiar with the word and its implications. In total there are 4 levels:

* newcomer
* basic
* advanced
* expert

So the word `'Asr` only needs to be clarified for a `newcomer` whereas from basic onward it does not need clarification